src/main/java/thaumicinsurgence/main/modules/planar_artifice/core/blocks/ItemBlockFurnaceDevice.java

```diff
@@ -0,0 +1,35 @@
+package thaumicinsurgence.main.modules.planar_artifice.core.blocks;
+
+import java.util.List;
+
+import net.minecraft.block.Block;
+import net.minecraft.creativetab.CreativeTabs;
+import net.minecraft.item.Item;
+import net.minecraft.item.ItemBlock;
+import net.minecraft.item.ItemStack;
+
+public class ItemBlockFurnaceDevice extends ItemBlock {
+
+    public ItemBlockFurnaceDevice(Block block) {
+        super(block);
+        setHasSubtypes(true);
+    }
+
+    @Override
+    public int getMetadata(int damageVal) {
+        return damageVal;
+    }
+
+    @Override
+    public void getSubItems(Item item, CreativeTabs tab, List list) {
+        // love the creative naming schemes mojang, thx XOXO
+        for (int i = 0; i < BlockAlkimiumAlchFurnace.blocks.length; i++) {
```

Comment:
this line was Furnace2 but that file didn't exist, and the existing one doesn't have "blocks"